apps/web/public/static/locales/en/common.json

```
@@ -4110,5 +4110,6 @@
   "timestamp": "Timestamp",
   "json": "JSON",
   "hubspot_ignore_guests": "Do not create new records for guests added to the booking",
+  "hubspot_create_event_on_contact": "Create event on contact, if it exists. Else fall back to lead",
```

```suggestion
"ignore_guests": "Do not create new records for guests added to the booking",
  "create_event_on_contact": "Create event on contact, if it exists. Else fall back to lead",
```

can you create keys without the app name, we will be adding the same features in other crm apps as well, we can just reuse it